server/index.js

```diff
@@ -18,5 +19,40 @@ app.get('/menu/:id', (req, res) => {
       res.send(menus)
     }
   })
-})
+});
+
+app.post('/menu', (req, res) => {
+  menus.create(req.params.id, (err, menus) => {
+    if (err) {
+      console.log(err)
```

Comment:
Remove stray console logs and replace it with res.status(400) with the error message